abis/linux/fsfilcnt_t.h

```diff
@@ -0,0 +1,6 @@
+#ifndef _ABIBITS_FSFILCNT_T_H
+#define _ABIBITS_FSFILCNT_T_H
+
+typedef __mlibc_uint64 fsfilcnt_t;
+
+#endif // _ABIBITS_FSFILCNT_T_H
```

Comment:
`abi-bits` are also public headers (ditto in the other dozen or so cases of line comments)